tests/test_project.cairo

```diff
@@ -625,31 +625,6 @@ fn fuzz_test_transfer_high_supply_high_amount(raw_supply: u256, raw_cc_amount: u
     );
 }
 
-// #[test]
```

Comment:
Will there be new tests for this, or it's not planned/useless?

Reply:
Good catch, this test isn't crucial but will add it in another PR.